templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -425,6 +424,126 @@ contents:
       return $s
     }
 
+    # Accepts parameters $iface_default_hint_file, $iface
+    # Writes content of $iface into $iface_default_hint_file
+    write_iface_default_hint() {
+      local iface_default_hint_file="$1"
+      local iface="$2"
+
+      echo "${iface}" >| "${iface_default_hint_file}"
+    }
+
+    # Accepts parameters $iface_default_hint_file
+    # Returns the stored interface default hint if the hint is non-empty,
+    # not br-ex, not br-ex1 and if the interface can be found in /sys/class/net
+    get_iface_default_hint() {
+      local iface_default_hint_file=$1
+      if [ -f "${iface_default_hint_file}" ]; then
+        local iface_default_hint=$(cat "${iface_default_hint_file}")
+        if [ "${iface_default_hint}" != "" ] &&
+           [ "${iface_default_hint}" != "br-ex" ] &&
+           [ "${iface_default_hint}" != "br-ex1" ] &&
+           [ -d "/sys/class/net/${iface_default_hint}" ]; then
+           echo "${iface_default_hint}"
+           return
+        fi
+      fi
+      echo ""
+    }
+
+    # Accepts parameters $bridge_name $bridge_interface
+    # Returns the physical interface name if $bridge_interface exists, "" otherwise
+    get_bridge_physical_interface() {
+      local bridge_interface="$1"
+      local physical_interface=""
+      nmcli conn show "${bridge_interface}" 1>/dev/null 2>&1
+      if [ $? -eq 0 ] ; then
+        physical_interface=$(nmcli -t --fields connection.interface-name conn show "${bridge_interface}" | awk -F ':' '{print $NF}')
```

Comment:
Using `nmcli -g connection.interface-name conn show "${bridge_interface}` might be cleaner here.

Reply:
done